solidity/contracts/Coordinator.sol

```diff
@@ -176,7 +176,7 @@ contract Coordinator is CoordinatorInterface {
   }
 
   bytes4 constant private permittedFunc =
-    bytes4(keccak256("executeServiceAgreement(address,uint256,uint256,bytes32,address,bytes4,bytes32,bytes)")); /* solium-disable-line indentation */
+    bytes4(keccak256("executeServiceAgreement(address,uint256,uint256,bytes32,bytes4,bytes32,bytes)")); /* solium-disable-line indentation */
```

Comment:
Interesting warning when updating this to use `.selector`:
```
Warning: Initial value for constant variable has to be compile-time constant. This will fail to compile with the next breaking version change.
  bytes4 constant private permittedFunc = this.executeServiceAgreement.selector;
                                          ^-----------------------------------^
```

Reply:
Seems like the selector is evaluated after the compilation phase, so you'd just have to remove the `constant` and I believe it should be okay.

Reply:
This resolves the warning, but creates a new storage slot for `permittedFunc`. Maybe we should just leave this line as-is until we decide to upgrade contracts to 0.5.0?